app/src/main/res/layout/fragment_add_login.xml

```diff
@@ -215,9 +217,7 @@
     <ImageButton
         android:id="@+id/clearPasswordTextButton"
         android:layout_width="48dp"
-        android:layout_height="30dp"
-        android:layout_marginTop="3dp"
-        android:layout_marginBottom="10dp"
+        android:layout_height="48dp"
         android:background="@null"
         android:contentDescription="@string/saved_logins_clear_password"
         android:visibility="invisible"
```

Comment:
outside the scope of this PR but noticed clearPasswordTextButton is not setting the same tint as clearUsernameTextButton